examples/xml/custom_source/settings.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<settings>
+
+  <run_mode>fixed source</run_mode>
+  <batches>10</batches>
+  <inactive>0</inactive>
+  <particles>100000</particles>
+
+  <!-- Starting source -->
+  <source>
+    <!--<library>./source_sample.so</library> -->
```

Comment:
```suggestion
```

Reply:
Sorry really confused by this change, do you want the line removed?

Reply:
I think this is resolved now